scalalib/src/mill/scalalib/ScalaModule.scala

```diff
@@ -171,7 +171,8 @@ trait ScalaModule extends JavaModule { outer =>
     resolveDeps(
       T.task {
         val bind = bindDependency()
-        Lib.scalaDocIvyDeps(scalaOrganization(), scalaVersion()).map(bind)
+        Lib.scalaDocIvyDeps(scalaOrganization(), scalaVersion()).map(_.withPlatformed(false)).map(bind)
+
```

Comment:
There is an extra empty line here

Reply:
If `scalaDocIvyDeps` always require `.withPlatformed(false)` we can move that to the definition instead, what do you think? Is it possible?